nuget.config.yaml

```diff
@@ -0,0 +1,8 @@
+author_string: Microsoft
+copyright_string: Copyright (c) Microsoft Corporation 2022
+description_string: Project Mu IPMI Feature Package.
+license_url: https://opensource.org/licenses/BSD-2-Clause

```

Comment:
should use https://opensource.org/licenses/BSDplusPatent to match spdx

Reply:
or maybe to match exactly since opensource.org doesn't have a perfect match.

https://spdx.org/licenses/BSD-2-Clause-Patent.html